numpy/core/src/umath/simd.inc.src

```diff
@@ -51,14 +52,33 @@ abs_ptrdiff(char *a, char *b)
      ((abs_ptrdiff(args[1], args[0]) >= (vsize)) || \
       ((abs_ptrdiff(args[1], args[0]) == 0))))
 
+/*
+ * Avoid using SIMD for very large step sizes for several reasons:
+ * 1) Supporting large step sizes requires use of i64gather/scatter_ps instructions,
+ *    in which case we need two i64gather instructions and an additional vinsertf32x8
```

Comment:
Tangentially related - it looks like the docs are incorrect from intel here:

* https://software.intel.com/en-us/cpp-compiler-developer-guide-and-reference-mm-i64gather-epi64-mm256-i64gather-epi64
* https://software.intel.com/en-us/cpp-compiler-developer-guide-and-reference-mm-i64gather-epi32-mm256-i64gather-epi32

Both these pages show the prototype as `extern __m128i _mm256...`, yet the documentation strongly indicates the return type is `__m256i`. Do you know the channel to report and correct this, since it appears you work for Intel?